src/lib/flashMessage.js

```diff
@@ -62,6 +62,18 @@ export function clearQueue(queueName) {
   Noty.closeAll(queueName);
 }
 
+/**
+ * Remove all messages from all queues without waiting for the animation
+ * to finish
+ */
+export function forceRemoveAll() {
```

Comment:
Since this is a new concept, it would be nice to mention it in the PR, what made you introduce it, and possibly for bonus points, to introduce it in a PR of it's own.

Reply:
You're right, I added a description for that